src/main/java/net/sacredlabyrinth/phaed/simpleclans/ui/frames/Components.java

```diff
@@ -114,7 +114,9 @@ public static SCComponent getClanComponent(@NotNull SCFrame frame, @NotNull Play
                     lang("gui.clandetails.clan.lore.allies", viewer, clan.getAllies().isEmpty() ? lang("none", viewer) : clan.getAllyString(lang("gui.clandetails.clan.lore.allies.separator", viewer))),
                     lang("gui.clandetails.clan.lore.rivals", viewer, clan.getRivals().isEmpty() ? lang("none", viewer) : clan.getRivalString(lang("gui.clandetails.clan.lore.rivals.separator", viewer))),
                     lang("gui.clandetails.clan.lore.founded", viewer, clan.getFoundedString()),
-                    lang("gui.clandetails.clan.lore.inactive", viewer, clan.getInactiveDays(), clan.isPermanent() ? "∞" : (clan.isVerified()) ? pl.getSettingsManager().getPurgeClan() : pl.getSettingsManager().getPurgeUnverified()));
+                    lang("gui.clandetails.clan.lore.inactive", viewer, clan.getInactiveDays(),
+                            clan.isPermanent() || (pl.getSettingsManager().getPurgeClan() <= 0 || pl.getSettingsManager().getPurgeUnverified() <= 0)
+                            ? "∞" : (clan.isVerified()) ? pl.getSettingsManager().getPurgeClan() : pl.getSettingsManager().getPurgeUnverified()));
```

Comment:
I think, there is should be better to create lang, which accepts Supplier<T>as argument and make it as lambda expression.